chat-plugins/modlog.js

```diff
@@ -145,12 +137,69 @@ function getMoreButton(room, search, useExactSearch, lines, maxLines) {
 	if (!newLines || lines < maxLines) {
 		return ''; // don't show a button if no more pre-set increments are valid or if the amount of results is already below the max
 	} else {
-		if (useExactSearch) search = Chat.escapeHTML(`"${search}"`);
 		return `<br /><div style="float:right"><button class="button" name="send" value="/modlog ${room}, ${search} ${LINES_SEPARATOR}${newLines}" title="View more results">More results...</button></div>`;
```

Comment:
is ``search`` escaped here at this point?

Reply:
ah forgot to just escape it nonetheless